### DISCUSSION TITLE
Does the one-line installation include everything?

### DISCUSSION THREAD

**Discussion by aliasboink** [StackStorm/st2#5695] [Q&A] [answered]
Hello, I am simply wondering if the one-line installation omits any possible feature of StackStorm. It does say it installs all components, but I am wondering if anything is missing from it when compared to the manual installations. 

Thank you!

**Comment by arm4b** [comment, marked as answer]
All-in-one installer provides a full experience and its logic is considered as a base for the other installers like Ansible, Puppet, Docker, K8s to be more or less consistent with the initial configuration and components included.
